translations/messages.pot

```diff
@@ -2626,11 +2654,11 @@ msgid ""
 "Base%(tag_close)s."
 msgstr ""
 
-#: octoprint_mrbeam/templates/mrbeam_doc_translation_placeholders.jinja2:1
+#: octoprint_mrbeam/templates/mrbeam_doc_translation_placeholders.jinja2:2
 msgid "Quickstart Guide"
 msgstr ""
 
-#: octoprint_mrbeam/templates/mrbeam_doc_translation_placeholders.jinja2:2
+#: octoprint_mrbeam/templates/mrbeam_doc_translation_placeholders.jinja2:3
 msgid "User Manual"
```

Comment:
There was a manual addition to the User Manual or Quickstart Guide a while back to resolve the MrbeamDoc issue. Is that still true? and if so, this might override the solution and might cause the issue again

Reply:
These changes are only a reference where the translated line was found.
What was this manual addition?? Do you have the PR or ticket?